EvernoteOAuth.lua

```diff
@@ -320,10 +320,6 @@ function OAuth:login()
     error("Target URL was not found in the response on login")
   end
 
-  if code == 302 then
-    return self:handleTwoFactor()
```

Comment:
Since you say it was never implemented, should the whole `handleTwoFactor` function be deleted?

Reply:
Actually yes, it can be completely removed